code/modules/mob/living/carbon/carbon.dm

```diff
@@ -441,6 +448,9 @@
 	if(has_trait(TRAIT_NOHUNGER))
 		return 1
 
+	if(!has_mouth())

```

Comment:
at the very least if you're not going ot make this a full organ, just add a TRAIT_NOMOUTH